cmd/network/governance/proposal.go

```diff
@@ -0,0 +1,66 @@
+package governance
+
+import (
+	"os"
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+)
+
+var propShowCmd = &cobra.Command{
+	Use:     "proposal-show <proposal-id>",
```

Comment:
This is strange, I would put this under the existing `show` not create a new subcommand?

Reply:
But it doesn't show the content?

Reply:
Yes, the current command does not show the content. Ok, I'll add content to the existing command. :)